lib/llm/src/model_card.rs

```diff
@@ -13,5 +13,17 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+use std::time::Duration;
+
 pub mod create;
 pub mod model;
+
+// TODO: Do these network/publish related model deployment card values belong here or in a
+// network module?
+
+/// Identify model deployment cards in the key-value store
+pub const BUCKET_NAME: &str = "mdc";
+
+/// Delete model deployment cards that haven't been re-published after this long.
+/// Cleans up if the worker stopped.
+pub const BUCKET_TTL: Duration = Duration::from_secs(5 * 60);
```

Comment:
What's the implication of a model card lying around? Say I have nats running in the background, I run dynamo, I kill dynamo, I run dynamo again later before this 5min TTL expires. What gotchas would I run into in between that TTL period?